modules/agar/src/main/ts/ephox/agar/api/FocusTools.ts

```diff
@@ -20,12 +20,12 @@ const cGetFocused: Chain<SugarElement<any>, SugarElement<any>> =
 const cSetFocused: Chain<SugarElement<any>, SugarElement<any>> =
   Chain.op(Focus.focus);
 
-const cGetOwnerDoc: Chain<SugarElement<Node>, SugarElement<Document>> =
-  Chain.mapper(Traverse.owner);
```

Comment:
I went through them when doing this PR and this should be all of them from what I could see.

Reply:
I should add there are many time `Traverse.owner` is what's wanted, it's mainly around focus or traverse down based cases. As an example, there's a case in `StickyHeader.ts` where the owner document is what we want, since we're trying to see if focus moved back to the body.